specification/azsadmin/resource-manager/deployment/Microsoft.Deployment.Admin/preview/2019-01-01/ActionPlan.json

```diff
@@ -0,0 +1,192 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01",
+    "title": "DeploymentAdminClient",
+    "description": "Deployment Admin Client."
+  },
+  "host": "adminmanagement.local.azurestack.external",
+  "schemes": ["https"],
+  "produces": ["application/json"],
+  "consumes": ["application/json"],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Deployment.Admin/locations/global/actionPlans": {
+      "get": {
+        "x-ms-examples": {
+          "Gets the list of action plans": {
+            "$ref": "./examples/ActionPlan/List.json"
+          }
+        },
+        "description": "Gets the list of action plans",
+        "tags": ["ActionPlans"],
+        "operationId": "ActionPlans_List",
+        "parameters": [
+          {
+            "$ref": "Deployment.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Deployment.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
```

Comment:
I think you need to add 'default' in each response, you could refer this as an example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/DatabaseSchema.json

Reply:
Prettier Check passed now. It was failing because I had to rebase from master.
@lirenhe Everything looks good now except the known avocado issue. We don't need the "default" value for each response. Could you please merge if it looks good otherwise? Thanks!